coderd/database/migrations/000261_improve_notification_templates.up.sql

```diff
@@ -0,0 +1,83 @@
+UPDATE notification_templates
+SET
+	body_template = REPLACE(body_template::text, 'Hi {{.UserName}}', 'Hi {{.UserName}},')::text
+WHERE
+	id = '29a09665-2a4c-403f-9648-54301670e7be';
```

Comment:
Have we allowed any customers to manually change these templates or mentioned it's a supported use-case? If yes, we should probably validate that the template hasn't been changed before updating it?

//cc: @dannykopping @mtojek

Reply:
Good question, but currently it is not supported 👍

Reply:
I did find the way we currently set and update these templates to be unintuitive and hard to work with. I have an idea that would make it easier to track changes to templates, but I'm not sure if it will be worth the complexity.

What I have in mind would introduce an auto incrementing "revision" column to the notification_templates table. This would make up and down migrations that adjust templates simpler, but it would make the queries that get the template in order to render it slightly more complex.

Reply:
This should no longer be an issue, because we've moved away from using REPLACE